pkg/console/install_panels.go

```diff
@@ -1384,7 +1384,23 @@ func addInstallPanel(c *Console) error {
 					return
 				}
 				logrus.Info("Local config (merged): ", c.config)
+
+				if strings.ToLower(c.config.VipMode) == config.NetworkMethodDHCP && (c.config.Vip == "" || c.config.VipHwAddr == "") {
+					if _, err := applyNetworks(c.config.Networks); err != nil {
```

Comment:
Maybe print `Configuring network" before applying? I remember this take some time.

Reply:
Updated it. Thank you.